fwprovider/provider.go

```diff
@@ -95,17 +98,6 @@ func (p *proxmoxProvider) Schema(_ context.Context, _ provider.SchemaRequest, re
 	resp.Schema = schema.Schema{
 		// Attributes specified in alphabetical order.
 		Attributes: map[string]schema.Attribute{
-			"api_token": schema.StringAttribute{
-				Description: "The API token for the Proxmox VE API.",
-				Optional:    true,
-				Sensitive:   true,
-				Validators: []validator.String{
```

Comment:
removed the `Validators` here as it was printing the "invalid" token to the logs (example if the format had a slight typo in it, ect) - moved the regex check down the chain of logic